bionlp/taggers/rnn_feature/tagger_utils.py

```diff
@@ -123,6 +123,41 @@ def trim_tags(tagged_data):
                           for x, y in tagged_data[i]]
     return tagged_data
 
+def get_regular_or_capitalized_embeddings(w2i, i2w, mdl, params):
+    embedding_vector_size = params['emb1_size']
+    embeddings = []
+    for i in range(len(w2i)):
+        term_emb = None
+        if i in i2w:
```

Comment:
You are correct. This is the same as what I stated in my previous comment: I wanted to change the original logic as little as possible. Originally, a list comprehension was used, and I changed it into a few separate if conditions since the logic got a little more complicated. I will adapt the logic accordingly.